zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1797,6 +1848,38 @@ components:
           description: The variables to complete the job with.
           type: object
           nullable: true
+    JobUpdateRequest:
+      type: object
+      properties:
+        changeset:
+          $ref: "#/components/schemas/JobChangeset"
+    JobChangeset:
+      description: |
+        JSON object with changed job attribute values.
+
+        The following attributes can be adjusted with this endpoint, additional attributes
+        will be ignored:
+
+        * `retries` - The new amount of retries for the job; must be a positive number.
+        * `timeout` - The duration of the new timeout in ms, starting from the current moment.
+
+        Providing any of those attributes with a null value or omitting it preserves the persisted attribute’s value.
+
+        The job cannot be completed or failed with this endpoint, use the Complete job or Fail job endpoints instead.
+      type: object
+      nullable: true
```

Comment:
❓ Does this imply that the request body can be left out? That's not what we want, right? It might be wrong, though, please check the OpenAPI spec again for reference.

Reply:
My bad 👍